tests/ipex/test_modeling.py

```diff
@@ -440,7 +440,7 @@ def test_langchain(self, model_arch):
 
 
 class IPEXLangchainTest(unittest.TestCase):
-    SUPPORTED_ARCHITECTURES = ("gpt2",)
```

Comment:
model from https://huggingface.co/Intel/tiny-random-gpt2 doesn't generate anything (same issue for original model, not specific to the ipex resulting model), replacing it with an other model